backend/database_handler/transactions_processor.py

```diff
@@ -78,14 +80,21 @@ def insert_transaction(
         data: dict,
         value: float,
         type: int,
+        nonce: int,
         leader_only: bool,
         client_session_id: str | None,
     ) -> int:
-        nonce = (
-            self.session.query(Transactions)
-            .filter(Transactions.from_address == from_address)
-            .count()
-        )
+        # Only allow transactions without from_address for 'fund_account'
```

Comment:
@denishacquin this type of validation would be better in the JSON RPC endpoint. It is the place where we are adding this types of validations so far.

Reply:
You're right - however, I was able to simply remove this check by passing the nonce explicitly from the fund_account endpoint method.

Note that it's weird to get a nonce from an empty address, but I guess this is just a hacky case we need to support for now to allow funding accounts, as later on we will probably be using faucets.